src/test/java/com/artipie/asto/fs/FileMetaTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/asto/LICENSE.txt
+ */
+package com.artipie.asto.fs;
+
+import com.artipie.asto.Meta;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.nio.file.attribute.FileTime;
+import java.time.Instant;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+/**
+ * Test case for {@link FileMeta}.
+ * @since 1.9
+ */
+final class FileMetaTest {
+    @Test
+    void readAttrs() {
+        final long len = 4;
+        final Instant creation = Instant.ofEpochMilli(1);
+        final Instant modified = Instant.ofEpochMilli(2);
+        final Instant access = Instant.ofEpochMilli(3);
+        final BasicFileAttributes attrs = Mockito.mock(BasicFileAttributes.class);
```

Comment:
is not it possible to create `BasicFileAttributes` instance without mockito?

Reply:
It's quite big interface with many attributes without public implementations (internal implementations depends on OS, e.g. `UnixFileAttributes` and `DosFileAttributes`). So instead of making fake classes with more than 10 methods (where we need only 3 methods) it looks easier to make mock in this case.